### DISCUSSION TITLE
Anker 4-Port USB 3.0 Hub, Ultra-Slim Data USB Hub Works

### DISCUSSION THREAD

**Discussion by mrmees** [mvp/uhubctl#476] [General]
Anker 4-Port USB 3.0 Hub, Ultra-Slim Data USB Hub with 2 ft Extended Cable [Charging Not Supported]

Confirmed all 4 ports work for power switching.  

https://www.amazon.com/dp/B07L32B9C2

**Comment by mvp** [comment]
Thanks for reporting! Can you please check if it supports vbus off - that is, connected phone stops charging or USB light stops shining?

**Comment by mrmees** [comment]
Yup, I checked phone charging on all 4 ports and was able to perform switching on and off.

**Reply by p-lappy** [reply]
Did you have to do anything special? I finished installing uhub and happened to alreaydy have the hub you linked.
I am able to disconnect my webcam just fine with either disconnecting it at the usb3.0 or usb2.0 "hub". But Ive tried turning off either level off and turning off all ports on both protocols and my USB light wont shut off.
I;'ve even unplugged the cam and plugged in usb light in the port and it turns on
